src/main/java/com/toptrumps/core/Game.java

```diff
@@ -1,4 +1,23 @@
 package com.toptrumps.core;
 
+import java.util.ArrayList;
+
 public class Game {
-}
+
+    private ArrayList<Player> players;
+
+    public Game() {
+        players = new ArrayList<Player>();
+    }
+
+    public void startNewGame(int numOfPlayers) {
+        for (int i = 0; i < numOfPlayers; i++) {
+            players.add(new AIPlayer("Player" + (i + 1)));
```

Comment:
IMO it's better to split this line into two for clarity:
1. Create a concatenated string with the player's name
2. Pass this string to the constructor

Reply:
agreed!

Reply:
Also, I'd avoid using truncating the method names... some times it's clear to you but it can cause some level of confusion for others. ```numOfPlayers``` → ```numberOfPlayers```. This is more like a code-styling issue, and all about trade-offs: longer method names in order to ensure clarity.